Sources/NIOCore/ByteBuffer-hex.swift

```diff
@@ -12,8 +12,30 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Foundation
```

Comment:
Is this allowed in NioCore? If not, can I use the "replaceWith" that is only MacOs 13.0 and above? Or should I implement the ability to replace white spaces of a string custom in this module?

Reply:
Ignore, found a solve.